packages/core/src/telemetry/metrics.ts

```diff
@@ -297,38 +484,27 @@ export function recordApiErrorMetrics(
   )
     return;
   const metricAttributes: Attributes = {
-    ...getCommonAttributes(config),
-    model,
-    status_code: statusCode ?? 'error',
-    error_type: errorType ?? 'unknown',
+    ...baseMetricDefinition.getCommonAttributes(config),
+    model: attributes.model,
+    status_code: attributes.status_code ?? 'error',
+    error_type: attributes.error_type ?? 'unknown',
```

Comment:
If possible, I feel the same commentary for `status_code` could potentially apply to `error_type`